config/tweaks.json

```
@@ -2680,6 +2680,11 @@
       $OneDrivePath = $($env:OneDrive)
       Write-Host \"Removing OneDrive\"
       $regPath = \"HKCU:\\Software\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\OneDriveSetup.exe\"
+      # Move OneDrive files to user profile root
+      if (Test-Path $OneDrivePath) {
+          Write-Host \"Copy downloaded Files from the OneDrive Folder to Root UserProfile\"
+          Start-Process -FilePath powershell -ArgumentList \"robocopy '$($OneDrivePath)' '$($env:USERPROFILE.TrimEnd())\\' /mov /e /xj\" -NoNewWindow -Wait
```

Hi! 😊 Nice idea. May i suggest creating a backup in a new folder instead of having everything from OneDrive be copied directly to user's home directory since doing this can make a lot of clutter in the home directory. For example:
```suggestion
          Start-Process -FilePath powershell -ArgumentList "robocopy '$OneDrivePath' '$($env:USERPROFILE.TrimEnd())\OneDriveBackup\' /mov /e /xj" -NoNewWindow -Wait
```